[Microsoft] Troubleshooting - auth-reset command (#1332)
demisto/content-docs: content-repo/extra-docs/articles/microsoft-auth-guide.md

| Microsoft Graph User | yes | yes - support both client secret and certificate | no | yes | yes |
| Microsoft Management Activity API (O365 Azure Events) | yes | no | no | yes | yes |
| Microsoft Teams | no | yes | no | yes | no |
| Microsoft Teams Management | yes | yes | yes | no | yes |

## Troubleshooting

> In case of errors in the authentication process - token revoked/expired or in case you generate new credentials, 

Reviewer: ```suggestion
In case of errors in the authentication process, such as a token revoked/expired or in case you generate new credentials, 
```

> you can use the `*!<integration command prefix>-auth-reset*` command in the War Rooms in order to rerun the authentication process,

Reviewer: ```suggestion
you can use the `*!<integration command prefix>-auth-reset*` command in the War Room in order to rerun the authentication process,
```

instead of recreating a new integration instance.

> Following verify the connectivity by clicking the Test button to test the instance.

Reviewer: ```suggestion
After running the command, click **Tes**t to verify the connectivity of the instance.
```

> For example, for Microsoft Graph Mail Single User,

Reviewer: ```suggestion
For example, when using the "self-deployed Azure app for Microsoft Graph Mail Single User, in case of an expired/revoked token error:
```

> When using the 'self-deployed Azure app':

Reviewer: ```suggestion

```

> In case of an expired/revoked token error, follow these steps:

Reviewer: ```suggestion

```
